fetch/api/resources/utils.js

```diff
@@ -44,6 +44,11 @@ function checkRequest(request, ExpectedValuesDict) {
   }
 }
 
+function encode_utf8(str)
+{
+    return unescape(encodeURIComponent(str));
```

Comment:
Although I hear that this works, I would feel more comfortable with `new TextEncoder.encode(str)` and comparing the resulting Uint8Arrays